src/cpp/client/credentials_cc.cc

```diff
@@ -21,12 +21,11 @@
 
 namespace grpc {
 
-static grpc::internal::GrpcLibraryInitializer g_gli_initializer;
```

Comment:
maybe remove constructors and destructors entirely from this cc file?

Reply:
Forgot to catch this. Updated.